rust/catalyst-types/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "catalyst-types"
-version = "0.0.5"
+version = "0.1.5"
```

```suggestion
version = "0.0.6"
```
Rust says: "0.0.x versions can be used for packages that are permanently unstable."

We should consider cat-libs crates permanently unstable until we are prepared to push to Crates.io which would mean we are `0.1.0`  for our first crates.io published version.
Actually under RUST rules, any `0.0.x` bump is a breaking change,